build-configs/configs/malte/index.js

```diff
@@ -3,10 +3,13 @@
 import { lightTheme, darkTheme } from '../../themes/malte'
 import type { BuildConfigType } from '../BuildConfigType'
 
+const APP_NAME = 'Malte'
+
 const MalteBuildConfig: BuildConfigType = {
-  appName: 'Malte',
+  appName: APP_NAME,
   lightTheme,
   darkTheme,
+  localesOverride: APP_NAME,
```

Comment:
Not sure whether we should identify what locales to use with something better :thinking: Proposals are welcome :)